tests/projz04000/Header

```diff
@@ -0,0 +1,22 @@
+HyperCLaw-V1.1
```

Comment:
@BenWibking Thanks for the question!

Yes, please do not check in binary files into the repository, to keep it small. We clone pyAMReX in many workflows, e.g., superbuilds and a slick repo size is important.

Could you potentially for the test generate the data on the fly, e.g., by running a generating function first that you read back?

Reply:
I am surprised the pre-commit.ci check `check-added-large-files` that we run on CI does not fail... concerning :-o

Reply:
My concern is that I don't know how to independently verify that the plotfile generation was correct if it's generated on the fly. Would it be possible to upload the test files as an extra .tar.gz file in a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases#storage-and-bandwidth-quotas), or to re-use the [test data on the yt website](https://yt-project.org/data/) under "boxlib frontend"?

Reply:
Downloading the test data on the fly is ok.
There is a downside to this, making some test runs slow and others impossible (w/o internet).

> My concern is that I don't know how to independently verify that the plotfile generation was correct if it's generated on the fly.

I think this can be verified once manually or in a separate test. (We also do not verify a downloaded files continuously, unless I misunderstand something here.)

Reply:
Ok, that makes sense. I think verifying the checksum for the generated plotfile would work instead of downloading a file (since that should be checksummed anyway).

Reply:
@BenWibking when you have the time, can you still update this? I would merge the PR once the large files are not checked in anymore :)

Reply:
I've removed this file, and generated the plotfile on-the-fly.